README.rst

```diff
@@ -111,3 +111,12 @@ See `COPYING <COPYING>`_ to see the full text.
    :target: https://app.shippable.com/projects/573f79d02a8192902e20e34b
 .. |Chat badge| image:: https://img.shields.io/badge/chat-IRC-brightgreen.svg
    :target: https://docs.ansible.com/ansible/latest/community/communication.html
+.. |Code Of Conduct| image:: https://img.shields.io/badge/Code%20of%20Conduct-Ansible-silver.svg
```

Comment:
Make the badge label lowercase since it's not a proper noun and that will match the other badges:

```suggestion
.. |Code Of Conduct| image:: https://img.shields.io/badge/code%20of%20conduct-Ansible-silver.svg
```